eng/common/TestResources/New-TestResources.ps1

```diff
@@ -215,11 +215,18 @@ try {
     $PSBoundParameters['BaseName'] = $BaseName
 
     # Try detecting repos that support OutFile and defaulting to it
-    if (!$CI -and !$PSBoundParameters.ContainsKey('OutFile') -and $IsWindows) {
+    if (!$CI -and !$PSBoundParameters.ContainsKey('OutFile')) {
         # TODO: find a better way to detect the language
```

Comment:
Re: `find a better way to detect the language`.

If we include `eng/common/scripts/common.ps1` you will be able to access the variable `$Language` which should have the correct language for the repo.